provider/ocm_policies_data_source.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift-online/ocm-sdk-go/logging"
+)
+
+const (
+	// Policy IDs from type operator roles
+	CloudCred       = "openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy"
+	CloudNetwork    = "openshift_cloud_network_config_controller_cloud_credentials_policy"
+	ClusterCSI      = "openshift_cluster_csi_drivers_ebs_cloud_credentials_policy"
+	ImageRegistry   = "openshift_image_registry_installer_cloud_credentials_policy"
+	IngressOperator = "openshift_ingress_operator_cloud_credentials_policy"
+	MachineAPI      = "openshift_machine_api_aws_cloud_credentials_policy"
+
+	// Policy IDs from type account roles
+	Installer            = "sts_installer_permission_policy"
```

Comment:
Hypershift clusters have a separate set of account and operator roles, I'd like to tag Ori on this as he's handling the work for it

Reply:
it sounds like for hypershift should separate the creation

Reply:
@gdbranco correct, for Hypershift we're going to have only three account roles: installer, support, and instance worker.

Also, the default flow for Hypershfit will use managed policies, and the policies ARNs should be sufficient.

Reply:
So can we resolve that conversation? @oriAdler @gdbranco ?